### ISSUE TITLE
PyPI package

### ISSUE THREAD

**Issue by ocervell** [laramies/theHarvester#1501]
Is there a way we can push theHarvester on PyPI to ease up the installation on automated environments ? A `pip install theHarvester` would be nice as an easy install method.

**Comment by NotoriousRebel** [comment]
Yes you are not wrong, please see #1435 for current progress on pipx and homebrew support. I assume once you can use pipx it should be trivial to create a formal pip package. Although just doing pip install will no longer be supported on [multiple OS's soon](https://www.kali.org/blog/python-externally-managed/) (ok well technically supported but highly frowned upon), so pipx, docker, virtualenv + pip, and homebrew may be sufficient enough.

**Comment by dbfreem** [comment]
What work would need to be done to make this work? This would probably help a lot of non Kali users

**Comment by L1ghtn1ng** [comment]
It’s something I have been meaning to take care of as everything is pretty much in place already for it to work. Planning on looking into getting this sorted on my time off in the very early part of December.

On 18 Nov 2023, at 01:33, Don Freeman ***@***.***> wrote:

﻿

What work would need to be done to make this work? This would probably help a lot of non Kali users

**Comment by ibrahmsql** [comment]
> Hi there, now that theHarvester is installable via pipx, can we release it to PyPI so we can just run: `pipx install theharvester` and it would work without cloning the repository ?

Hi yep you can now install theHarvester directly using pip or pipx. Enjoy
